mirdata/utils.py

```diff
@@ -103,24 +102,46 @@ def check_index(dataset_index, data_home, silence=False):
 
 
 def validator(dataset_index, data_home, silence=False):
-    """validate.. (todo: what does it do?) """
-    missing_files, invalid_checksums = check_index(dataset_index, data_home, silence)
+    """logs missing files and files with invalid checksums in a dataset index.
```

Comment:
Maybe we should elaborate this idea a bit more? I'm not sure if a new user will understand it otherwise. Something like "checks the existence and/or validity of files stored locally with respect to the paths and file checksums stored in the reference index, and logs missing or invalid files." Just an idea

Reply:
thank you. fixed in 5df7c5f